Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml

```diff
@@ -1717,6 +1717,18 @@ UseGPUProcessForWebGLEnabled:
       "PLATFORM(WIN)": true
       default: false
 
+UserActivationAPIEnabled:
+   type: bool
+   humanReadableName: "User Activation API"
+   humanReadableDescription: "Enable User Activation API"
+   defaultValue:
+     WebKitLegacy:
+       default: false
+     WebKit:
+       default: false
+     WebCore:
+       default: false
+
```

Comment:
What's the plan for enabling this feature? Anything blocking it?

Reply:
It's not really a blocker but: some developers are confused that this feature means that "user activation model v2" of HTML is implemented everywhere throughout webkit. I've tried to explain that that is not the case and this is a very poor proxy for making any such assumption. 

Apart from that, there are a couple of satellite discussions happening in WPT I'd like to finalize before we enable it.